frontend/src/v5/ui/routes/viewer/groups/groupItem/groupItemContainer.component.tsx

```diff
@@ -57,7 +56,7 @@ export const GroupsTreeListItemComponent = (
 		<GroupsTreeListItemContainer $depth={depth}>
 			{children}
 			<ButtonsContainer>
-				{!isV5() && <TooltipButton action={onClickIsolate} label="Isolate" Icon={Visibility} disabled={disabled} />}
+				<TooltipButton action={onClickIsolate} label="Isolate" Icon={Visibility} disabled={disabled} />
```

Comment:
This should be removed as the isV5 is negated. This causes the styling to look weird.

![image](https://github.com/3drepo/3drepo.io/assets/62481250/b620fdb0-d471-4144-a913-38a2e3ba1432)

(staging)
![image](https://github.com/3drepo/3drepo.io/assets/62481250/51b9358b-1174-4e08-ae98-0c1d4733bc10)